common/app/model/Cors.scala

```diff
@@ -8,11 +8,11 @@ object Cors extends Results with implicits.Requests {
 
   private val defaultAllowHeaders = List("X-Requested-With","Origin","Accept","Content-Type")
 
-  def apply(result: Result, allowedMethods: Option[String] = None, fallbackAllowOrigin: Option[String] = None)(implicit request: RequestHeader): Result = {
+  def apply(result: Result, allowedMethods: Option[String] = None, fallbackAllowOrigin: Option[String] = None, extraWhitelist: List[String] = Nil) (implicit request: RequestHeader): Result = {
```

Comment:
Is there a test for this that needs updating now?

Reply:
Will update tests in a following PR. 👍